From 629446cc695a91b84ec21413632dc84ee326b2e7 Mon Sep 17 00:00:00 2001
From: Alex Zhondin <lexxmark.dev@gmail.com>
Date: Sat, 8 Sep 2018 22:32:26 -0700
Subject: [PATCH] fix include paths for cmake

---
 bison/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/bison/CMakeLists.txt b/bison/CMakeLists.txt
index 02fad2c..05e3ee8 100644
--- a/bison/CMakeLists.txt
+++ b/bison/CMakeLists.txt
@@ -18,5 +18,6 @@ add_executable(${PROJECT_NAME}
 )

 target_include_directories(${PROJECT_NAME} PRIVATE "src")
+target_include_directories(${PROJECT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}")

 target_link_libraries(${PROJECT_NAME} common_lib kernel32.lib user32.lib)
--
